tck-runner/pom.xml

```diff
@@ -323,5 +320,47 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>security-manager</id>
+            <activation>
+                <property>
+                    <name>with-security-manager</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <properties>
+                <arquillian.protocol>LocalSecurityManagerTesting</arquillian.protocol>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.hibernate.beanvalidation.tck</groupId>
+                    <artifactId>beanvalidation-standalone-container-adapter</artifactId>
+                    <version>${tck.version}</version>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <systemProperties>
+                                <property>
+                                    <name>validation.provider</name>
+                                    <value>${validation.provider}</value>
+                                </property>
+                                <property>
+                                    <name>excludeIntegrationTests</name>
+                                    <value>true</value>
+                                </property>
+                            </systemProperties>
+                            <argLine>-Djava.security.manager -Djava.security.policy=${project.build.directory}/test-classes/test.policy -Djava.security.debug=access</argLine>
```

Comment:
> So the difference to the 'local' profile is the arg line enabling the security manager and the custom arquillian protocol, right?

Right.

> Did you think about just modifying the local profile to run with security manager and not even provide a switch. Would there be a drawback? On the plus side I can see less lines of config code.

Interesting idea. Being able to run the tests without the SM may still come in handy when analysing bugs, test failures etc. as there is one less moving part. I've changed it to one profile, running with or without the SM being enabled based on the property. The default is now running with the SM, but it can be disabled like so:

    mvn clean install -pl tck-runner/ -Dwith-security-manager=false

> Also what's about mvn clean install -Dincontainer -Dwith-security-managet, that would not make sense, right?

One actually could think about running the tests on WildFly with the SM enabled. But I won't add it for the time being as it functionally should be the same.

> If possible we should try to make it impossible ('local' and 'inconainer' are mutual exclusive ) or at least make appropriate notes in the readme.

I'll add a note.